platzky/blog/blog.py

```
@@ -1,10 +1,16 @@
+import logging
 from os.path import dirname
+from typing import Any, Callable
```

Ruff (0.14.10)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)